### DISCUSSION TITLE
How do I get the twind intellisense extension to work with fresh in vscode?

### DISCUSSION THREAD

**Discussion by michael-pfister** [freshframework/fresh#616] [Q&A] [answered]
I initialized a new fresh project and chose the option to initialize it with twind. After that I got the 'Twind Intellisense' extension for VScode as recommended. However, I'm not getting any autocompletion in my project whatsoever. Not sure what's wrong here I tried to get this to work for hours now.

Seems like I'm not the only one having this issue #317 had a couple of comments regarding the same question. Any help would be greatly appreciated!

![image](https://user-images.githubusercontent.com/63919069/184399770-13d51aad-4220-478b-86c4-ccabb4a6e9af.png)
![image](https://user-images.githubusercontent.com/63919069/184399784-5e17d9b3-8542-488a-aa2a-c61a89a1642f.png)

**Comment by nnmrts** [comment, marked as answer]
You need to add a file called `twind.config.js` to your project root folder, then it works. The file can be empty, but you can of course actually use it as well, just gotta import and include it inside the `main.ts` file. Reference: https://github.com/tw-in-js/vscode-twind-intellisense/issues/9

**Reply by michael-pfister** [reply]
![image](https://user-images.githubusercontent.com/63919069/184685503-47d2b331-63c6-4c2e-91eb-61879cf32839.png)

omg thank you so much I was stuck for hours with this
